contrib/scalar/scalar.c

```diff
@@ -116,6 +116,14 @@ static int set_recommended_config(void)
 		{ "core.multiPackIndex", "true" },
 		{ "core.preloadIndex", "true" },
 		{ "core.safeCRLF", "false" },
+#ifdef HAVE_FSMONITOR_DAEMON_BACKEND
+		/*
+		 * The value proposition of Scalar is really only realized by
+		 * using FSMonitor. We choose to only support the built-in
+		 * FSMonitor, and only enable this on supported platforms.
+		 */
+		{ "core.useBuiltinFSMonitor", "true" },
```

Comment:
We should probably turn this off on `scalar unregister`, right?

Reply:
Our philosophy has been to not disable settings as we cannot detect which ones were explicitly chosen by the user.

Reply:
Okay.